db/migrate/20190329151023_add_rid_to_stages.rb

```diff
@@ -0,0 +1,7 @@
+class AddRidToStages < ActiveRecord::Migration[5.1]
+  def change
+    add_column :stages, :rid, :string
```

Comment:
This name is not meaningful and will be very confusion to others.  Maybe `secure_access_key` or `randon_access_key`.  The name should be easily identifiable.